docs/get-started/typescript.md

```diff
@@ -17,178 +17,122 @@ Before you can compile and run the target project, you will need to initialize i
 
 > **Note:** you can use an existing project if you have one, in that case, you can skip the following section.
 
-Execute the following command in the directory you want to create a new project.
+Execute the following commands in the directory where you want to create a new project.
 
-```shell
-git clone https://github.com/FreekMencke/node-typescript-starter.git
-npm i
+```bash
+npm init
+npm install -D typescript ts-node
+npx tsc --init
 ```
 
-> **Note:** in this example the `node-typescript-starter` is used, but you can use any node + typescript template.
-
 ## Adding dependencies
 
 ### Getting access to the packages
 
-If you have not already, you will need to create a `.npmrc` to enable access to the packages in the GitHub package feed.
+> **Note:** This is a temporary requirement while Kiota is in preview.
 
-1. Add a `.npmrc` file with the following content
+1. Navigate to [New personal access token](https://github.com/settings/tokens/new) and generate a new token. (permissions: `package:read`, `repo`).
+1. Copy the token, you will need it later.
+1. Enable SSO for the Microsoft org on the token if you are a Microsoft employee.
+1. Add a new file named **.npmrc** in the root of the project with the following content.
 
     ```conf
     @microsoft:registry=https://npm.pkg.github.com/
     ```
 
-1. `npm login --scope=@microsoft --registry=https://npm.pkg.github.com` (use a token with package:read, repo and SSO enabled for the Microsoft organization as the password)
-1. `npm i @microsoft/kiota-authentication-azure -S`.
+1. Run the following command to authenticate NPM to the GitHub package server. When prompted, enter your GitHub username and use the personal access token you generated earlier as the password.
 
-### Installing Kiota
+    ```bash
+    npm login --scope=@microsoft --registry=https://npm.pkg.github.com
+    ```
 
-The article on installing the [Kiota command line](./index.md) tool shows how to do this.
+### Add Kiota and dependencies
 
 Once the package feed is accessible the following packages can be added to the project.
 
-```shell
-npm i @microsoft/kiota-abstractions -S
-npm i @microsoft/kiota-authentication-azure -S
-npm i @microsoft/kiota-http-fetchlibrary -S
-npm i @microsoft/kiota-serialization-json -S
-npm i @azure/identity node-fetch
+```bash
+npm install @microsoft/kiota-abstractions
+npm install @microsoft/kiota-authentication-azure
+npm install @microsoft/kiota-http-fetchlibrary
+npm install @microsoft/kiota-serialization-json
+npm install @azure/identity node-fetch
```

Comment:
```suggestion
npm install @azure/identity node-fetch@2.X
```

last time I tested 3.X didn't work, if it does ignore this comment.

Reply:
It seemed to work fine for me, but of course I'm only doing a `GET /me`. Was there something specific that didn't work?

Reply:
request wasn't getting through at all (some misalignment of APIs), but if you say 3.X now works, perfect! (I have locked to 2.X in my test projects back then). Resolving